embedded/devices/src/mppt.cpp

```diff
@@ -0,0 +1,249 @@
+#include "mppt.h"
+#define ID 0x0000
```

Comment:
Let's rename this macro to be a little more descriptive. My first thought is "MPPT_BASE_ID"